pylorcon2/PyLorcon2.c

```diff
@@ -22,7 +22,7 @@
 */
 
 
-#include <python2.7/Python.h>
+#include <python3.6/Python.h>
```

Comment:
Please do not hardcode any Python version.
Just use `#include <Python.h>`.
`setup.py` already passes `-I/usr/include/pythonX.Y` for appropriate Python version.